README.md

```diff
@@ -52,6 +52,14 @@ To contribute to the project, please try to use pull-requests and follow our [co
 Unit test status:
 [![xpra](https://github.com/Xpra-org/xpra/actions/workflows/test.yml/badge.svg)](https://github.com/Xpra-org/xpra/actions/workflows/test.yml)
 
+## GNOME Shell Extension
+
+The `input-source-manager@xpra_org` GNOME Shell extension (see `gnome_shell_extension/` directory)
+must be installed if it is desired to update the platform keyboard layout when the `next_keyboard_layout` is called.
+This extension may be installed by `/usr/bin/gnome-extensions enable input-source-manager@xpra_org` command
+or alternatively, it may be enabled manually by searching for the `Extensions` (by pressing the `Super` key)
+and finding that extension in the filtered list.
+
```

Comment:
This does not belong on the home page!

Reply:
I removed this part and integrated its information with the README.md file which exists in `fs/share/gnome-shell/extensions/input-source-manager@xpra_org/` directory.